[availability] (pre-release) Replacing preview with information about flighted roll outs (#454)
OfficeDev/office-scripts-docs: docs/includes/rollout-note.md

> [!NOTE]

> > This feature is being rolled out and is not available to everyone. It is slowly released to larger numbers of users to ensure things are working smoothly. This feature is subject to change based on your feedback.

Reviewer: I think "things are working smoothly" is an ambiguous phrasing that's difficult to localize. Perhaps: 
```suggestion
> This feature is being rolled out and is not available to everyone. It's slowly released to larger numbers of users to ensure that it's working as expected. This feature is subject to change based on your feedback.
```
